develop/tutorials/articles/220-internationalization/01-localizing-your-application.markdown

```diff
@@ -170,7 +170,8 @@ to the module, and registers a
 [ResourceBundleLoader](@platform-ref@/7.0-latest/javadocs/portal-kernel/com/liferay/portal/kernel/util/ResourceBundleLoader.html)
 service for that capability. A capability is a contract a module declares to
 @product@'s OSGi framework. It lets you associate services with modules that
-provide them. 
+provide them. `ResourceBundleLoaderAnalyzerPlugin` is the plugin that performs
+the detection. 
```

Comment:
Right, Blade users are using the ResourceBundleLoaderAnalyzerPlugin without knowing it.